modules/core/rapid_context.js

```diff
@@ -22,13 +35,55 @@ export function coreRapidContext(context) {
                     if (maxlat === undefined || lat > maxlat) maxlat = lat;
                     if (maxlon === undefined || lon > maxlon) maxlon = lon;
                 }
+
+                if (f.geometry.type === 'LineString' && lineStringCount === 1) {
+                    var lats = f.geometry.coordinates.map((f) => {return f[0];});
```

Comment:
These anonymous fat arrow functions can be written a bit more concisely..
With braces you can have multiple statements but must supply an explicit return.. Without braces, you have a single expression with an implicit return. 

```suggestion
                    var lats = f.geometry.coordinates.map(f => f[0]);
```

Reply:
Ooh, good catch. Also fixed the not-quite-es6ified filter & map calls on lines 49 and 51.